web/src/components/RemoteContent.tsx

```diff
@@ -8,19 +8,34 @@ const SandBox = styled.div<{ centered: boolean }>`
   font-family: ${props => props.theme.fonts.web.contentFont};
   font-size: ${props => props.theme.fonts.contentFontSize};
   line-height: ${props => props.theme.fonts.contentLineHeight};
+
   ${props =>
     props.centered &&
     css`
       text-align: center;
       list-style-position: inside;
     `}
-
   & img {
     max-width: 100%;
     max-height: 100%;
     object-fit: contain;
   }
 
+  & figure {
+    margin-inline-start: 0;
+    text-align: center;
+    margin: 15px auto;
+    @media only screen and (max-width: 640px) {
+      width: 100% !important;
+    }
+  }
+
+  & figcaption {
+    font-size: ${props => props.theme.fonts.figCaptionFontSize};
+    font-style: italic;
```

Comment:
Any specific reason why you're differing in the styles on web and native?

Reply:
no just forgot to set it italic on native. thx for the hint